content/posts/2019-02-msbuild-global-properties-defineconstants.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Learning about MSBuild Global Properties - The Hard Way"
+date = "2019-02-25"
+description = "Learning about MSBuild Global Properties - The Hard Way"
+categories = [ "Development", "Stack Overflow", "MSBuild", "FAIL" ]
++++
+
+One of the first steps in the Stack Overflow .NET Core migration was to extract a Common library that would target `netstandard`.
```

Comment:
Worth noting we're not really targeting `net462`/`netcoreapp` here and using the TFM as a split which is true for our case but not universally useful